dk.tangramgames.mrrescue.json

```diff
@@ -0,0 +1,93 @@
+{
+  "app-id": "dk.tangramgames.mrrescue",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "mrrescue",
+  "finish-args": [
+    "--device=all",
+    "--socket=pulseaudio",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "PhysicsFS",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2",
+          "sha256": "b77b9f853168d9636a44f75fca372b363106f52d789d18a2f776397bf117f2f1"
+        }
+      ]
+    },
+    {
+      "name": "libmodplug",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/modplug-xmms/files/libmodplug/0.8.9.0/libmodplug-0.8.9.0.tar.gz",
+          "md5": "5ba16981e6515975e9a68a58d5ba69d1"
+        }
+      ]
+    },
+    {
+      "name": "luajit",
+      "buildsystem": "simple",
```

Comment:
This shouldn't be using `simple`; set `no-autogen: true` and `make-install-args: ["PREFIX=/app"]`.

You also need the `luajit` beta for `aarch64` support.

Reply:
> You also need the luajit beta for aarch64 support.

I was unaware of that.
Would it be preferable to use the beta version or to stick with 2.0.5 and blacklist aarch64 until 2.1.0 goes stable?